src/protocol/mod.rs

```diff
@@ -1029,6 +1040,13 @@ mod serialize {
         }]);
         roundtrip(Protocols::new(vec![protocol]))
     }
+
+    #[test]
+    fn includes_the_environment() -> R<()> {
+        let mut protocol = Protocol::empty();
+        protocol.env.insert("FOO".to_string(), "BAR".to_string());
```

Comment:
`FOO` is the environment variable and `BAR` is the value? If so, I'd do `bar` instead for the value.

Reply:
Done.